deploy/kubernetes/driver/kubernetes/manifests/node-server.yaml

```diff
@@ -36,9 +36,9 @@ spec:
                 command: ["/bin/sh", "-c", "rm -rf /registration/vpc.file.csi.ibm.io /registration/vpc.file.csi.ibm.io-reg.sock"]
           env:
             - name: ADDRESS
-              value: /csi/csi.sock
+              value: /csi/csi-vpc-file.sock
```

Comment:
verify and run this node service as non-root i.e update runAsUser: 1000, runAsNonRoot: false is also not required ---> next PR is also fine